src/main/java/org/fcrepo/importexport/exporter/Exporter.java

```diff
@@ -129,6 +135,8 @@ public Exporter(final Config config, final FcrepoClient.FcrepoClientBuilder clie
                 bagdir.mkdirs();
                 this.bag = new Bag(new Version(0, 97));
                 this.bag.setRootDir(bagdir);
+                this.bag.getMetadata().putAll(bagConfig.getBagInfo());
+                //TODO where to add aptrust bag info from the bagConfig?
```

Comment:
Yes, for reference, that's what BagWriter is doing: https://github.com/LibraryOfCongress/bagit-java/blob/master/src/main/java/gov/loc/repository/bagit/writer/BagWriter.java#L255